### DISCUSSION TITLE
Support for Ryzen 55600g

### DISCUSSION THREAD

**Discussion by coimbrox** [ChefKissInc/NootedRed#114] [General]
Hey guys sorry for my question, but i dont see any comment about support on this Chipset, a few months ago they dont have, now is ok?

**Comment by trueight** [comment]
im trying this cpu but still unsuccess, when i put noot kext im stuck on a black screen

**Reply by CraigHazan** [reply]
How much VRAM are you giving it in BIOS?

**Reply by MatitaPere** [reply]
I'm having the same issue. I've already put 1gb in Vram, tried several SMBIOS. Can't boot into the OS (10.15 here...). I know Catalina is not yet  100% supported, mabybe I should wait a little longer?

**Reply by CraigHazan** [reply]
Have you tried a different macOS version?, might be worth trying Big Sur at least. I don't think Catalina is fully supported yet, there have been recent updates on the 'Actions' page.

**Reply by MatitaPere** [reply]
Yes, in big Sur and Ventura it works fine. I just need it to work in Catalina because I got an audio interface that only works up to 10.15. Hopefully it will be supported soon

**Reply by CraigHazan** [reply]
Someone responded to one of my threads on tonymacx86 in regards to the 5600G. 
_Yes, 5600g work fine with nootedred when VDA decoder is disabled (nred default)), if AppleGVA is enabled by boot arg -nredvcn, some 3rd apps like IINA, VLC, h264 and 8 bit HEVC video playback may be failed to initiate video surface (10 bit HEVC video playback is OK), others seems to be OK._
Hope this helps.

**Reply by MatitaPere** [reply]
I assume this is for Big Sur/Ventura right?

**Reply by CraigHazan** [reply]
![1690041510313](https://github.com/NootInc/NootedRed/assets/20331487/51354419-4e41-44a7-9aef-ee700ad86c69)

**Comment by Lmororo97** [comment]
im stuck on a black screen after the install of monterey

**Comment by VisualEhrmanntraut** [comment]
55600g -> 5 5600g. That's 5XXX. CPU support is in our site. https://nootinc.github.io/nred

**Comment by lucasolegario** [comment]
Is the graphics part performing well? I'm having a lot of lag and crashes @ChefKissInc @CraigHazan

**Reply by CraigHazan** [reply]
I think so, on the A320I-K with the Ryzen 5 4600G and the Ryzen 7 4700G, I was impressed and surprised. Finder looks great, animations look smooth. No Man's Sky plays well, it's a big step up from UHD 630 from Intel.

**Reply by lucasolegario** [reply]
Thanks for the feedback, the system is too slow I can't use it, my current hardware is A520-M PRO VH + Ryzen 5 5600g + 16GB Ram 3600mhz

**Reply by CraigHazan** [reply]
You should be getting decent acceleration on that setup. Have you tried changing the amount of VRAM in BIOS?.

**Reply by lucasolegario** [reply]
I set a 4gb VRAM in the bios, but I noticed that when booting the system it says my current VRAM is 7mb

**Reply by lucasolegario** [reply]
![WhatsApp Image 2023-07-31 at 22 27 49](https://github.com/NootInc/NootedRed/assets/45582797/e991145e-50fe-4b36-977a-6c4d27ee1894)

**Reply by CraigHazan** [reply]
You don't have acceleration working yet, doesn't look like NRed loaded.

**Reply by lucasolegario** [reply]
Is there anything I can do to get NRed loaded? would reinstalling be an option?

**Reply by CraigHazan** [reply]
I'm on tonymacx86, AMD OS X and the Discord, ideally it would be good if we can get a look at your EFI on there, then others can also contribute. You haven't got WhateverGreen in your kexts have you?, it's one or the other.

**Reply by VisualEhrmanntraut** [reply]
@lucasolegario You need to update your other kexts

**Reply by lucasolegario** [reply]
Thank you for the help, I will try to update my kexts! @ChefKissInc

**Reply by lucasolegario** [reply]
I was able to reinstall using Monterey and the graphics are working now, thank you very much for your time! @CraigHazan @ChefKissInc

**Comment by MatitaPere** [comment]
Tried latest nred with catalina support. Still a black screen before loading OS. Any suggestion? Thanks

**Reply by lucasolegario** [reply]
Change SMBIos and update update your EFI to last versions of kexts

**Reply by MatitaPere** [reply]
I had tried different SMBios before, I'll see if there are kexts updates. thanks!

**Reply by MatitaPere** [reply]
no luck so far

**Reply by MatitaPere** [reply]
If I use latest Catalina support master I get black screen. If I use latest HMDI I get panic 
![IMG_6088](https://github.com/NootInc/NootedRed/assets/8464526/b895151b-12e2-4381-958f-71e6855078d8)
